src/main/java/org/folio/service/impl/RequestEventHandler.java

```diff
@@ -201,6 +213,13 @@ private void propagateChangesFromPrimaryToSecondaryRequest(EcsTlrEntity ecsTlr,
       secondaryRequestId, secondaryRequestTenantId);
 
     boolean shouldUpdateSecondaryRequest = false;
+    if (primaryRequest.getStatus() == Request.StatusEnum.CLOSED_CANCELLED) {
+      log.info("propagateChangesFromPrimaryToSecondaryRequest:: primary request is cancelled, " +
+        "cancelling secondary request");
+      secondaryRequest.setStatus(Request.StatusEnum.CLOSED_CANCELLED);
```

Comment:
I think we should cancel secondary request using mod-circulation's API, not mod-circulation-storage's